### DISCUSSION TITLE
Borg backup not working 500 internal server error

### DISCUSSION THREAD

**Discussion by lukalagebr** [nextcloud/all-in-one#4827] [Questions] [answered]
**Steps to reproduce**
I just set up the AIO container using a docker compose file inside dockge

**Expected behavior**
backup works as expected

**Actual behavior**
500 Internal Server Error log message 

**Host OS**
Truenas Scale Dragonfish-24.04.1.1
I did Jip-Hop jailmaker tutorial and installed dockge to set up AIO container

**Nextcloud AIO version**
v8.3.0

**Current channel**
latest

Hi thank you guys in advance for the help. First time posting here so please just let me know if I made any mistakes.

So I have Nextcloud AIO installed and running but I am not being able to run the Borg Backup it gives me an error message. It is attached my nextcloud AIO log and also my compose file that I used to install Nextcloud AIO I am still learning linux and docker by my self so it is possible to have something wrong in my configurations but I tried to search here and google and did not find a solution yet.

Thanks.

[Nextcloud AIO log.txt](https://github.com/user-attachments/files/15829423/Nextcloud.AIO.log.txt)
[Nextcloud aio config.txt](https://github.com/user-attachments/files/15829424/Nextcloud.aio.config.txt)

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#failure-of-the-backup-container-in-lxc-containers

**Reply by lukalagebr** [reply]
Hi, Thank you for your answer and to take your time to look at it, and sorry for the late reply. I was trying to figure this out. I spent some time reading this readme that you sent , I did install fuse in my host that is a Truenas Scale and I also installed fuse inside the Jailmaker but when I run modprobe fuse inside the jailmaker shell I get this message "modprobe: FATAL: Module fuse not found in directory /lib/modules/6.6.29-production+truenas" if I run modinfo fuse I get this one "modinfo: ERROR: Module fuse not found." my jailmaker is running debian12 and I use Dockge to manage my containers. I am kind of lost I just starting to learn docker and I never heard about fuse since I got this error message. I am attaching a screenshot of the shell with the errors. I am not sure if I installed fuse right I just used apt-get install fuse command.  And again thank you very much for your help and patience.
![shell truenas and docker](https://github.com/nextcloud/all-in-one/assets/172670599/01b1ac58-8dbd-41b1-b449-da41fa73e5d7)

**Reply by realkev86** [reply]
Hi, install fuse in your docker shell: 

```
sudo apt-get update
sudo apt-get install fuse
```

Then in your stack/container settings add necessary capabilities and device access (here for docker compose)

    cap_add:
      - SYS_ADMIN
    devices:
      - /dev/fuse:/dev/fuse
